library/ix-dev/charts/elastic-search/questions.yaml

```diff
@@ -0,0 +1,176 @@
+groups:
+  - name: Elastic Search Configuration
+    description: Configure Elastic Search
+  - name: User and Group Configuration
+    description: Configure User and Group for Elastic Search
+  - name: Network Configuration
+    description: Configure Network for Elastic Search
+  - name: Storage Configuration
+    description: Configure Storage for Elastic Search
+  - name: Resources Configuration
+    description: Configure Resources for Elastic Search
+
+portals:
+  web_portal:
+    protocols:
+      - "$kubernetes-resource_configmap_portal_protocol"
+    host:
+      - "$kubernetes-resource_configmap_portal_host"
+    ports:
+      - "$kubernetes-resource_configmap_portal_port"
+    path: "$kubernetes-resource_configmap_portal_path"
+
+questions:
+
+  - variable: esConfig
+    label: ""
+    group: Elastic Search Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: password
+          label: Password
+          description: The password for the Elastic Search (elastic) user.
+          schema:
+            type: string
+            default: ""
```

Comment:
JFYI defaults do not need to be specified for attrs which are `required` and user should be specifying values for them always